app/app.js

```diff
@@ -21,41 +19,12 @@ app.use(compression());
 app.use(express.json({ limit: config.get('server.bodyLimit') }));
 app.use(express.urlencoded({ extended: true }));
 
-// Logging Setup
-log.level = config.get('server.logLevel');
-log.addLevel('debug', 1500, { fg: 'cyan' });
-
-let logFileStream;
-let teeStream;
-if (config.has('server.logFile')) {
-  // Write to logFile in append mode
-  logFileStream = fs.createWriteStream(config.get('server.logFile'), { flags: 'a' });
-  teeStream = new Writable({
-    objectMode: true,
-    write: (data, _, done) => {
-      process.stdout.write(data);
-      logFileStream.write(data);
-      done();
-    }
-  });
-  log.disableColor();
-  log.stream = teeStream;
-}
-
 // Print out configuration settings in verbose startup
 log.verbose('Config', utils.prettyStringify(config));
```

Comment:
These pre-existing log statements need to be reformatted - for this one specifically, we should not be pretty stringifying the config contents anymore as winston already handles object printing well. I'd consider either changing this to the following, or outright removing it (no function attribute as this is a top level action; set to **debug** as this is likely to print out sensitive secrets to console...)
```js
// Print out configuration settings in debug startup
log.debug('App configuration', config);
```